db/repos/rounds.js

```diff
@@ -88,11 +88,11 @@ class RoundsRepository {
 	 * Update the votes of for a particular account.
 	 *
 	 * @param {string} address - Address of the account
-	 * @param {int} amount - Votes to update
+	 * @param {BigNumber} amount - Votes to update
```

Comment:
I think it will be more simple to test/maintain the functions if we would always pass `string` into it and convert it to BigNumber when needed. It would also simplify the possible external tools basing on JsDOC to create documentation/typing etc.

Reply:
No, there it was decided to maintain `Bignumber` instances whatever possible and convert to `string` when needed (API, save to database).

Reply:
Yes, since we keeping bignum over the application layer, whenever its required like frontend or database it can be converted to the string, this way we are 100% sure we are operating with bignum all over the application.

Reply:
Ok then :)